Product/models.py

```diff
@@ -0,0 +1,45 @@
+from django.db import models
```

Comment:
```suggestion

from django.db import models
```
모듈 임포트 이후엔 한 줄 띄워주세요